{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'ai.praktika.android' application. Consider checking the driver's troubleshooting documentation. Original error: Cannot start the 'ai.praktika.android' application. Consider checking the driver's troubleshooting documentation. Original error: Activity name '.ai.praktika.android.MainActivity' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity \nHost info: host: 'fv-az1756-999', ip: '10.1.0.60'\nBuild info: version: '4.22.0', revision: 'c5f3146703*'\nSystem info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.5.0-1023-azure', java.version: '21.0.3'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:appActivity: ai.praktika.android.MainAct..., appium:appPackage: ai.praktika.android, appium:autoGrantPermissions: true, appium:automationName: UiAutomator2, appium:noReset: true, platformName: ANDROID}]}]\nCapabilities {appium:appActivity: ai.praktika.android.MainAct..., appium:appPackage: ai.praktika.android, appium:autoGrantPermissions: true, appium:automationName: UiAutomator2, appium:noReset: true, platformName: ANDROID}","children":[{"name":"setUp","uid":"c14c3a7667e13bcc","parentUid":"04b9b49c8b1b94c9b7997413266b98f6","status":"broken","time":{"start":1720999339390,"stop":1720999339391,"duration":1},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["Android","local"],"tags":[]}],"uid":"04b9b49c8b1b94c9b7997413266b98f6"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}